### ISSUE TITLE
Incorrect Tags

### ISSUE THREAD

**Issue by jasoniej** [ansible-lockdown/Windows-2019-CIS#52]
18.4.11: tagged as level 1, should be level 2
18.5.10.2: tagged as level 1, should be level 2
18.8.22.1.2: tagged as level 1, should be level 2

**Comment by georgenalen** [comment]
@jasoniej,
 I looked at 18.5.10.2 and it's set to level2 in the tag, was there a different control you meant to enter in there? I want to make sure we catch all of the missed tags. 

-George

**Comment by jasoniej** [comment]
@georgenalen
below is what I see in both devel/main (the tags are level1-* not level2-*):

```
- name: "SCORED | 18.5.10.2 | PATCH | L2 Ensure Turn off Microsoft Peer-to-Peer Networking Services is set to Enabled"
  win_regedit:
      path: HKLM:\Software\Policies\Microsoft\Peernet
      name: Disabled
      data: 1
      type: dword
  when:
      - rule_18_5_10_2
  tags:
      - level1-domaincontroller
      - level1-memberserver
      - rule_18.5.10.2
      - patch
```

**Comment by georgenalen** [comment]
@jasoniej,
 Thanks for confirming, that must have been fixed with the start of the updates for 1.2.1 because it is correct in that update branch. All of those fixes will be merged into devel/main in the next couple of weeks

-George
